### PR TITLE
improve documentation regarding external dependencies

### PR DESCRIPTION
**Pull request by CaptTofu**
I had some problems on OS X that I was chasing my tail with. I realize now what I was doing wrong and updated the docs to reflect an incorrect include path as well as missing golang packages. 

many thanks to Travis Cline for the help!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,12 +36,14 @@ make check
 sudo make install
 ```
 
-Then, `go get -u` as usual.
+Then, `go get -u` as usual the following packages:
 
 ```sh
+go get -u github.com/golang/protobuf/protoc-gen-go
+go get google.golang.org/grpc
```
Comment (achew22) [inline README.md]: Can you add a -u to this call to go get

File: README.md [hunk 2]
```diff
@@ -36,12 +36,14 @@ make check
...
 go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
 go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
-go get -u github.com/golang/protobuf/protoc-gen-go
+go get -u golang.org/x/net/context`
```
Comment (achew22) [inline README.md]: I think there is a stray ` at the end of this line

File: README.md [hunk 3]
```diff
@@ -89,7 +91,7 @@ Make sure that your `$GOPATH/bin` is in your `$PATH`.
    ```sh
    protoc -I/usr/local/include -I. \
      -I$GOPATH/src \
-     -I$GOPATH/src/github.com/googleapis/googleapis/ \
+     -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis
```
Comment (achew22) [inline README.md]: Can you add a trailing \ on this line?

File: README.md (hunk 2 above)
Comment (CaptTofu) [inline README.md]: will fix :)

File: README.md (hunk 1 above)
Comment (CaptTofu) [inline README.md]: will do!

File: README.md (hunk 3 above)
Comment (CaptTofu) [inline README.md]: yes

**Comment by CaptTofu** [comment]
Made changes. Also, I am under HPE's Google Corporate CLA

File: README.md [hunk 4]
```diff
@@ -36,12 +36,14 @@ make check
 sudo make install
 ```
 
-Then, `go get -u` as usual.
+Then, `go get -u` as usual the following packages:
 
 ```sh
+go get -u github.com/golang/protobuf/protoc-gen-go
```
Comment (tmc) [inline README.md]: Why are these additions necessary?

Comment (tmc) [inline README.md]: I accept the first one, for the others added (grpc,context) I would prefer to tell users to 'go get .' after generating code.

Comment (CaptTofu) [inline README.md]: The reason I chose explicit packages is I kept adding whatever the results of "go build" would complain about such as in "cannot find package "google.golang.org/grpc"

I didn't know about "go get .". I'll just simply use that as I tested that it works.

**Comment by CaptTofu** [comment]
Hi there - I think I made all the changes that were requested. Also, I'm covered under the CLA - not sure why that is giving a "status to be reported"

**Comment by CaptTofu** [comment]
I signed it!

**Pull request merged**